src/pages/_document.tsx

```diff
@@ -39,8 +40,14 @@ export default class MyDocument extends Document {
           <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon/favicon-32x32.png" />
           <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon/favicon-16x16.png" />
 
-          {/* Plausible Analytics */}
-          <script defer data-domain="safe.global" src="https://plausible.io/js/script.js"></script>
+          {IS_PRODUCTION && (
```

Comment:
```suggestion
          {IS_PRODUCTION && IS_OFFICIAL_HOST && (
```

Need we also add this to the `ContentSecurityPolicy`?

Reply:
AFAICS, we have neither this variable nor CSP in this repo.

Reply:
Very true, I thought we were in another one 🙈